### ISSUE TITLE
Backup Configurations Update

### ISSUE THREAD

**Issue by s7thamon** [laravel/forge-sdk#119]
According to docs there's no way to update the databases selected in an existing Backup Configuration? There's an option in the web gui to do this, at least with Provider set as Custom (S3 Compatible). You can edit and include additional databases.

This would be great to have for automated provisioning scripts.

**Comment by jbrooksuk** [comment]
It is possible to update the databases within a backup configuration now, so I've updated the documentation to reflect this.

The SDK can now be updated to do this.

**Comment by s7thamon** [comment]
I ran `$forge->updateBackupConfiguration` with the existing ID and database list and it wiped out the fields that were not entered such as endpoint, bucket, etc. No errors were returned. On the Web GUI they were all empty and now unable to enter them back in:

![image](https://user-images.githubusercontent.com/66268528/119718381-7f1bbb00-be35-11eb-9a44-b56d1f0fc103.png)

**Comment by s7thamon** [comment]
I noticed when using `$forge->updateBackupConfiguration` and only specifying the ID and Databases in the data array, it wiped out all of the other information in the GUI such as the endpoint, bucket, keys, etc.

Editing them in the GUI after threw errors despite fields being entered. I had to delete the configuration and re-create it.

The SDK did not throw any errors at the time.

![image](https://user-images.githubusercontent.com/66268528/119721831-99f02e80-be39-11eb-96c3-c941e58d752e.png)
